### DISCUSSION TITLE
Text shadow

### DISCUSSION THREAD

**Discussion by ggolda** [SixLabors/ImageSharp.Drawing#151] [Q&A]
Is there a way to create a shadow for a text? The only thing that I found is to use something similar described in this issue: https://github.com/SixLabors/ImageSharp/issues/407 and create a border. But shadow have blur and other parameters that I didn't figure out how to setup using this border approach.

**Comment by JimBobSquarePants** [comment]
Draw it once on a blank image, blur that result, draw it again at an offset then draw that image onto your target. That how all libraries would do it.
